app/models/course.rb

```diff
@@ -260,7 +260,8 @@ def self.entry_requirement_options_without_nil_choice
   validates :subjects, presence: true, on: :publish
   validate :validate_enrichment_publishable, on: :publish
   validate :validate_site_statuses_publishable, on: :publish
-  validate :validate_provider_visa_sponsorship_publishable, on: :publish
+  validate :validate_provider_visa_sponsorship_publishable, on: :publish, if: -> { recruitment_cycle.year.to_i >= 2022 }
+  validate :validate_provider_ukprn_publishable, on: :publish, if: -> { recruitment_cycle.year.to_i >= 2022 }
```

Comment:
See 
https://github.com/DFE-Digital/teacher-training-api/pull/2016/files#r662111263

Reply:
I played around with using a constant but came up short on a good name. Also experiemented with having a method on `RecruitmentCycle` instead of have to do `.year.to_i` all the time. Again, struggling to come up with a good method name. Happy to follow the route suggested.

Reply:
@darokel and I had a conversation about this yesterday and eventually decided that `recruitment_cycle.next_cycle?` was something we could live with. `next_cycle?` will always be true for all future cycles no matter what year we're in.